apps/ledger-live-mobile/src/components/CustomImage/useCenteredImage.tsx

```diff
@@ -102,6 +102,7 @@ function useCenteredImage(params: Params) {
           );
         })
         .catch(e => {
+          if (dead) throw new Error("dead hook");
```

Comment:
we agree that this `Error("dead hook")` will be catched just below in the `load().catch(e => ...)` and nothing will happen ? If this error is actually bubbled up, this error message is not enough descriptive

Reply:
yep nothing happens, this is done for typing